pkg/minikube/constants/constants.go

```diff
@@ -39,4 +39,14 @@ func MakeMiniPath(fileName string) string {
 	return filepath.Join(Minipath, fileName)
 }
 
+var LogFlags = [...]string{
+	"logtostderr",
+	"alsologtostderr",
+	"stderrthreshold",
+	"logdir",
+	"logbacktraceat",
+	"v",
+	"vmodule",
```

Comment:
Are all these flags required to register on both minikube and localkube, or are they for internal use?